src/EventStore.ClientAPI/Projections/ProjectionsClient.cs

```diff
@@ -142,9 +142,10 @@ public Task<string> GetQuery(EndPoint endPoint, string name, UserCredentials use
             return SendGet(endPoint.ToHttpUrl(httpSchema, "/projection/{0}/query", name), userCredentials, HttpStatusCode.OK);
         }
 
-        public Task UpdateQuery(EndPoint endPoint, string name, string query, UserCredentials userCredentials = null, string httpSchema = EndpointExtensions.HTTP_SCHEMA)
+        public Task UpdateQuery(EndPoint endPoint, string name, string query, UserCredentials userCredentials = null, string httpSchema = EndpointExtensions.HTTP_SCHEMA, bool emitEnabled = true)
```

Comment:
OK, so you wanna make this method accepts bool?, and if not set it will be execlude emit paramater from Url, is that right ?